ivy/functional/backends/numpy/creation.py

```diff
@@ -137,7 +137,11 @@ def full_like(
     device: str,
     out: Optional[np.ndarray] = None,
 ) -> np.ndarray:
+ numpy_sort
```

Comment:
Hi! There seem to still be a few missed lines with unwanted text. Please kindly remove these lines and commit the changes. Thanks!